.github/workflows/publish-containers.yml

```diff
@@ -43,21 +50,35 @@ jobs:
           if [[ $VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
             # Matches 1.0.0, 1.0.1, etc.
             NAMED_VERSION="latest"
+
+            # setting version for cache-from
+            IFS="." read -r -a VERSION_ARRAY <<< "$VERSION"
+            if [ "${VERSION_ARRAY[-1]}" -gt 0 ]; then
+              PREVIOUS_VERSION[-1]="$((${VERSION_ARRAY[-1]} - 1))"
+              printf -v CACHE_FROM_VERSION "%s." "${PREVIOUS_VERSION[@]}"
+              CACHE_FROM_VERSION="${CACHE_FROM_VERSION%.}"
+            else
+              # setting to unstable-main if patch is lower than 1
+              CACHE_FROM_VERSION="unstable-main"
+            fi
           else
             # Matches 1.0.0a1, 1.0.0rc1, etc.
             NAMED_VERSION="snapshot"
+            CACHE_FROM_VERSION="unstable-main"
           fi
 
           TAGS=$"\
-          ghcr.io/${{ steps.image.outputs.image_name }}:${NAMED_VERSION},\
-          ghcr.io/${{ steps.image.outputs.image_name }}:${VERSION}\
+          ghcr.io/${{ steps.image.outputs.image_name }}:${NAMED_VERSION}-buildcache,\
+          ghcr.io/${{ steps.image.outputs.image_name }}:${VERSION}-buildcache\
           "
 
           # Output the target tags
           echo "
-          CONTAINER_TAGS=${TAGS}
```

Comment:
This change will be reverted, is is only to prevent pushing version that could affect the users during tests